website/docs/docs/cloud/manage-access/scim.md

```diff
@@ -27,32 +41,60 @@ If your IdP isn’t on the list, it can be supported using dbt Cloud APIs (_docs
 
 Please complete the [setup SSO with Okta](/docs/cloud/manage-access/set-up-sso-okta) steps before configuring SCIM settings.
 
-To configure the SCIM settings for Okta:
+
+### Set up dbt Cloud
+
+To retrieve the necessary dbt Cloud configurations for use in Okta:
 
 1. Navigate to your dbt Cloud **Account settings**.
 2. Select **Single sign-on** from the left-side menu.
-3. Click **Edit**, scroll to the bottom of your Okta configuration settings, and click **Enable SCIM**.
-    - In later steps, you will need both the **SCIM base URL** and **SCIM token**.
+3. Scroll to the bottom of your Okta configuration settings and click **Enable SCIM**.
+    <Lightbox src="/img/docs/dbt-cloud/access-control/enable-scim.png" width="60%" title="SCIM enabled in the Okta configuration settings." />
+4. Record the **SCIM base URL** field for use in a later step.
+5. Click **Create SCIM token**.
+6. In the pop-out window, give the token a name that will make it easily identifiable. Click **Save**.
+    <Lightbox src="/img/docs/dbt-cloud/access-control/name-scim-token.png" width="60%" title="Give your token and identifier." />
+7. Copy the token and record it securely, as _it will not be available again after you close the window_. You must create a new token if you lose the current one. 
+    <Lightbox src="/img/docs/dbt-cloud/access-control/copy-scim-token.png" width="60%" title="Give your token and identifier." />
+8. (Optional) Manual updates are turned off by default for all SCIM-managed entities, including the ability to invite new users manually. This ensures SCIM-managed entities stay in sync with the IdP, and we recommend keeping this setting disabled.
```

Comment:
do we recommend keeping this disabled or enabled? in the [notion doc](https://www.notion.so/dbtlabs/SCIM-Beta-Documentation-173bb38ebda78046a9bfeb7a8ff40c74?pvs=4#1babb38ebda780e3b96df332459b992f), i'm seeing it says to keep it enabled so it stays in insync?

Reply:
It's worded strangely in the notion doc for sure lol. This text here is what it should say